treeherder/seta/preseed.py

```diff
@@ -20,11 +58,20 @@ def load_preseed():
     The fields buildtype, testtype and platform can have * which makes ut match  all flavors of
     the * field. For example: (linux64, pgo, *) matches all Linux 64 pgo tests
     """
-
+    logger.debug("START")
+    # XXX: Caching of runnable jobs is not working very well use global variable for now
```

Comment:
Somehow the caching of `get_reference_data_names` is not working properly and I need to make use of `global` in here so I can have access within `validate_preseed_entry()`.